PowerEditor/src/WinControls/TabBar/TabBar.cpp

```diff
@@ -777,6 +776,8 @@ LRESULT TabBarPlus::runProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lPara
 			int currentTabOn = getTabIndexAt(xPos, yPos);
             if (_isDragging)
 			{
+				_isDragging = false;

```

Comment:
This line is not part of modification of:
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/4886/files

Any explanation?

Reply:
> I could never replicate [#4885](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4885). I figured it's probably related to ReactOS and/or WINE. A test of Npp++ on ReactOS revealed a very similar bug which I've patched here as "_Speculative fix for Issue#4885_"

Reply:
There's **two** commits in this PR. The first reverts PR#4886. The second includes a fix for a bug I noticed in ReactOS which may also be the cause of #4885 (which PR#4886 allegedly fixes).

Reply:
In this case, please create a new PR which contains only the revert of the commit in question:
`git revert c6e1a95098ada4db56dd9032bd5b2d4fa92458a7` 

And then create the second PR which could fix the issue https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4885